cxplat/inc/winuser/cxplat_winuser.h

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+#pragma once
+#include <winerror.h>
+
```

Comment:
I'd also recommend:
```c
#define CXPLAT_FAILED(X) FAILED(X)
#define CXPLAT_SUCCEEDED(X) SUCCEEDED(X)
```

Reply:
I dislike having two macros that do the same thing.  How is FAILED different from !SUCCEEDED?

Reply:
98% of the time, code uses the failed marco for error conditions, so if we did have to pick just 1, I'd prefer that. But I do like having two for the cases where you want the opposite. Having english words instead of just ! the other is nicer (IMO) to read.